testsuite/tests/parsing/dropped_attributes.ml

```diff
@@ -0,0 +1,15 @@
+(* TEST
+   ocamlc_byte_exit_status = "2";
+   setup-ocamlc.byte-build-env;
+   ocamlc.byte;
+   check-ocamlc.byte-output;
+*)
+
+(* Attributes [[@annot1]] and [[@annot3]] have not node in the parsetree
```

Comment:
I don't think this test covers `annot3`. Maybe we need another test to do so? But I think probably an expect-test will work nicely here.

Reply:
How do you make an expect test work with something that's a syntax error?

Reply:
Good point.

Reply:
Made an additional file to demonstrate both attributes.

Reply:
Combined the two tests back into one file using "toplevel", which lets you do this!